main.js

```diff
@@ -0,0 +1,89 @@
+const statusDisplay = document.querySelector('.game--status');
+
+let gameActive = true;
+let currentPlayer = "X";
+let gameState = ["", "", "", "", "", "", "", "", ""];
+
+const winningMessage = () => `Player ${currentPlayer} has won!`;
+const drawMessage = () => `Game ended in a draw!`;
+const currentPlayerTurn = () => `It's ${currentPlayer}'s turn`;

```

Comment:
These don't need to be functions you could have your messages just equal those strings without the fat arrow syntax e.g. `const winningMessage = 'Player ${currentPlayer} has won!` When you use them later you would then not need to call them, but could just use the variable name.